src/app/store/transactions/token-transfer.hooks.ts

```diff
@@ -183,21 +183,10 @@ export function useGenerateFtTokenTransferUnsignedTx(selectedAssetId: string) {
 }
 
 // TODO: Refactor when remove legacy send form?
-export function useFtTokenTransferUnsignedTx(
-  selectedAssetId: string,
-  values?: StacksSendFormValues
-) {
-  const generateTx = useGenerateFtTokenTransferUnsignedTx(selectedAssetId);
+export function useFtTokenTransferUnsignedTx(assetBalance?: StacksFungibleTokenAssetBalance) {
+  const generateTx = useGenerateFtTokenTransferUnsignedTx(assetBalance);
```

Comment:
@fbwoolf input here, re comment. does this still need to be removed?

Reply:
I think just the comment can be removed now, but hook should remain to use for calculating fees.